index.html

```diff
@@ -299,6 +299,35 @@ <h2>Contact me</h2>
     <hr class="contact-hr">
   </section>
 </section>
+
+<div class="modal-container">
+  <div class="modalbox">
+    <div class="pop-top">
+      <div class="top-title">
+        <h3 class="modaltitle"></h3>
+        <img onclick="CloseModal()" class="close-modal" src="images/close.jpg" alt="closebutton">
+    </div>
+    <div class="project_info">
+        <p class="pop-company"></p>
+      <ul class="ul">
+        <li class="pop-role"></li>
+        <li class="pop-year"></li>
+    </ul>
+    </div>
+  </div>
+    <img class="popup-image" alt="Project Popup Image">
+    <div class="below-box">
+    <p class="pop-up-description"></p>
+    <div class="sec-box">
+      <ul class="language_used_modal"></ul>
+    <div class="modal-button-div">  
+  <a  class="seelive"><button type="button">See live <img class="button-image" src="images/export.svg" alt="live"> </button></a>    
+  <a  class="seesource"><button type="button">See source  <img class="button-image" src="images/github.png" alt="github"></button></a>  
+    </div>
```

Comment:
- [ ] The requirement is that the modal popup should be created dynamically and not in the index.html  file